### PR TITLE
Add trial details.

### PR DESCRIPTION
**Pull request by chadwcarlson**
## Why

Closes https://github.com/platformsh/platformsh-docs/issues/3810

## What's changed

Adds Upsun trial details.

### CODE REVIEW

File: sites/friday/src/get-started/here/setup.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ While the guide has been written to accommodate the following stacks, it is not
 
 {{% guides/requirements %}}
 
+{{% note theme="info" title="Trials" %}}
+When you create an account on Upsun, you are also beginning a trial for your organization.
+[See the documentation](/glossary#trial) for more details about limits to that trial.
```
Comment (AnouckColson) [inline sites/friday/src/get-started/here/setup.md:45]:
```suggestion
When you create an account on Upsun, you are also beginning a trial for your organization.
Get [more information on trials](/glossary#trial).
```

File: sites/friday/src/glossary/_index.md [hunk 2]
```diff
@@ -201,3 +201,24 @@ TLS is the successor of Secure Socket Layer (SSL).
 It provides the cryptographic "S" in HTTPS.
 It's often still referred to as SSL
 even though it has mostly replaced SSL for online encrypted connections.
+
+## Trial
+
+When you create an organization for the first time on Upsun, you will have time and resources available so you can try out the Upsun platform.
+It has come restrictions that are relevant to your time experimenting with the platform:
```
Comment (AnouckColson) [inline sites/friday/src/glossary/_index.md:208]:
```suggestion
When you create an account on Upsun, you are also beginning a trial for your organization.
This trial grants you a certain amount of time and resources to try out the Upsun platform, and comes with the following restrictions:
```

File: sites/friday/src/glossary/_index.md [hunk 3]
```diff
@@ -201,3 +201,24 @@ TLS is the successor of Secure Socket Layer (SSL).
...
+1. A trial lasts for **5 days**, which starts automatically from creating your first organization.
+1. A trial allows for **one (1)** active [project](#project) at a time.
+1. For a trial project, the trial allows for **two (2)** [active environments](#active-environment).
+1. At the organization level, there is a limit on the amount of [resources available to your project](/manage-resources):
+
+    - 4.5 CPU
+    - 12 GB RAM
+    - 20 GB storage
+
+If for some reason your trial ends before adding payment details, there are some additional things to keep in mind:
```
Comment (AnouckColson) [inline sites/friday/src/glossary/_index.md:219]:
```suggestion
If your trial ends before adding payment details, there are additional things to keep in mind:
```

File: sites/friday/src/glossary/_index.md [hunk 4]
```diff
@@ -201,3 +201,24 @@ TLS is the successor of Secure Socket Layer (SSL).
...
+
+- When a trial expires, both the organization and the project are suspended.
+- Projects where no code has been pushed to them are deleted **1 day** after a trial expires. 
+- Projects where code _has_ been pushed to them are deleted **5 days** after the trial expires.
```
Comment (AnouckColson) [inline sites/friday/src/glossary/_index.md:223]:
```suggestion
- Projects where no code has been pushed are deleted **1 day** after a trial expires. 
- Projects where code _has_ been pushed are deleted **5 days** after the trial expires.
```

File: sites/friday/src/security/data-retention.md [hunk 5]
```diff
@@ -47,3 +47,8 @@ These "tombstone" backups are retained for between 7 days and 6 months depending
 
 {{% vendor/name %}} uses Google Analytics on various web pages, and so Google Analytics stores collected data for a period of time.
 We have configured our Google Analytics account to store data for 14 months from the time you last accessed our site, which is the minimum Google allows.
+
+## Trials
+
+User data - which includes pushed code and data contained within services - is retained for a shorter period during trials.
+See [Trial details](/glossary#trial) for more information.
```
Comment (AnouckColson) [inline sites/friday/src/security/data-retention.md:54]:
```suggestion
See [Trial details](/glossary#trial). 
```

**Review by AnouckColson — DISMISSED** [review]

**Review by AnouckColson — APPROVED** [review]

File: sites/friday/src/get-started/here/setup.md [hunk 6]
```diff
@@ -40,6 +40,11 @@ While the guide has been written to accommodate the following stacks, it is not
 
 {{% guides/requirements %}}
 
+{{% note theme="info" title="Trials" %}}
+When you create an account on Upsun, you are also beginning a trial for your organization.
```
Comment (tylers-username) [inline sites/friday/src/get-started/here/setup.md:44]:
```suggestion
When you create your first organization on Upsun, you are also activating your trial for that organization.
```

File: sites/friday/src/glossary/_index.md [hunk 7]
```diff
@@ -201,3 +201,24 @@ TLS is the successor of Secure Socket Layer (SSL).
 It provides the cryptographic "S" in HTTPS.
 It's often still referred to as SSL
 even though it has mostly replaced SSL for online encrypted connections.
+
+## Trial
+
+When you create an account on Upsun, you are also beginning a trial for your organization.
```
Comment (tylers-username) [inline sites/friday/src/glossary/_index.md:207]:
```suggestion
When you create your first organization on Upsun, you are also activating your trial for that organization.
```

**Review by tylers-username — COMMENTED** [review]
A couple of suggestions. Open to feedback 😄

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**